lib/driver-cli.js

```diff
@@ -0,0 +1,265 @@
+import _ from 'lodash';
+import { KNOWN_DRIVERS, validateDriversJson, writeInstalledDriver,
+         INSTALL_TYPE_LOCAL, INSTALL_TYPE_NPM, INSTALL_TYPE_GIT,
+         INSTALL_TYPE_GITHUB } from './drivers';
+import { getLatestVersion, installPackage, linkPackage } from './npm';
+import semver from 'semver';
+
+let INSTALLED_DRIVERS;
+
+function errAndQuit (json, msg) {
+  if (json) {
+    console.log(JSON.stringify({error: msg}));
+  } else {
+    console.error(msg.red);
+  }
+  process.exit(1);
+}
+
+function log (json, msg) {
+  !json && console.log(msg);
+}
+
+async function runDriverCommand (args) {
```

Comment:
I would be happy if functions could have docstrings with parameters and returns types descriptions

Reply:
I promise to add them once the actual methods and parameters are approved.